From 4c335948cf98730bace18adf40f908cd1ae3627f Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Thu, 08 Oct 2020 12:31:50 +0300
Subject: [PATCH] [SW-14199] make: pre-move repos under embedded

some projects use buildroot-external folder as a relative path to specs
we have SPECS_ROOT environment variable in habana_env file
this commit change these projects to use SPECS environment variable

Change-Id: I3389fba7128064641df0ad703d50a37019f2472b
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
---

diff --git a/Makefile b/Makefile
index 410404d..910af4a 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,10 @@
 CC = $(CROSS_COMPILE)gcc
 OBJCOPY = $(CROSS_COMPILE)objcopy
 SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(SPECS_DEV)/asic_reg
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
-CFLAGS += -include $(dir $(BR2_EXTERNAL))/embedded/specs/$(SPECS_DEV)/$(SPECS_DEV)_reg_map.h
+CFLAGS += -I$(SPECS_DIR)
+CFLAGS += -I$(SPECS_DIR)/$(SPECS_DEV)/asic_reg
+CFLAGS += -I$(SPECS_EMBEDDED_DIR)
+CFLAGS += -include $(SPECS_EMBEDDED_DIR)/$(SPECS_DEV)/$(SPECS_DEV)_reg_map.h
 CFLAGS += -Wall -Werror
 CFLAGS += -g -c
 CFLAGS += -I ./include/ -I ./ -I ./common/ -I ./$(HL_DEV)/
diff --git a/recovery/Makefile b/recovery/Makefile
index 34764e8..9bbd5ca 100644
--- a/recovery/Makefile
+++ b/recovery/Makefile
@@ -1,8 +1,8 @@
 CC = $(CROSS_COMPILE)gcc
 OBJCOPY = $(CROSS_COMPILE)objcopy
 SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(SPECS_DEV)/asic_reg
+CFLAGS += -I$(SPECS_EMBEDDED_DIR)
+CFLAGS += -I$(SPECS_DIR)/$(SPECS_DEV)/asic_reg
 CFLAGS += -Wall -Werror
 CFLAGS += -g -c
 CFLAGS += -I ./../include/ -I ./../ -I ./../common/ -I ./../$(HL_DEV)/
